ios/Podfile.lock

```diff
@@ -10,13 +10,13 @@ PODS:
   - qr_code_scanner (0.2.0):
     - Flutter
     - MTBBarcodeScanner
-  - Sentry (7.0.3):
```

Comment:
is it common that Apps commit lock files? pod and pubspec, feels like always a bit harder to bump versioning

Reply:
Yep, for applications (ie consuming software) you'll always want to commit lock files so that every developer works with the same versions. You'll end up with less `it works on my machine` mistakes.

Reply:
but it always works on my machine 🗡️ 
makes sense, thanks